src/olympia/devhub/tests/test_views_versions.py

```diff
@@ -431,6 +431,19 @@ def test_rejected_can_request_review(self):
         assert buttons.length == 1
         assert buttons.text() == u'Request Review'
 
+    def test_webextension_cannot_request_review(self):
+        self.addon.update(status=amo.STATUS_NULL)
+        latest_version = self.addon.find_latest_version(
+            channel=amo.RELEASE_CHANNEL_LISTED)
+        for file_ in latest_version.files.all():
+            file_.update(is_webextension=True)
```

Comment:
Add `status=amo.STATUS_DISABLED` as the test above does, so it's the exact same state, apart from being a webextension.